src/frontend/src/app/flow/palette/filter.pipe.ts

```diff
@@ -6,7 +6,9 @@ import { Pipe, PipeTransform } from '@angular/core';
 export class FilterPipe implements PipeTransform {
   transform(items: any[] | undefined, term: string): any[] | undefined {
```

Comment:
Items zijn toch van het type JSON?

Reply:
It gest converted with result.json() to a Promise<any. I could make a defenition file / model in the future.